packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -57,11 +63,26 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
       });
 
       this.webCallingService = new WebCallingService(this.$webex, this.$config.callingClientConfig);
+      this.taskManager = TaskManager.getTaskManager(
+        this.services.contact,
+        this.webCallingService,
+        this.services.webSocketManager
+      );
 
+      this.incomingTaskListener();
       LoggerProxy.initialize(this.$webex.logger);
     });
   }
 
+  /**
+   * An Incoming Call listener.
+   */
+  private incomingTaskListener() {
```

Comment:
We should be switching off all listeners on the logout function being called right? Applies to other listeners in WebCallingService too. Additionally, we should also deregister on the calling SDK?

Reply:
@Kesari3008 but deregister doesn't take care of these listeners right? This is a listener on taskManager.